File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.py
def solution(order):
    menu = {"iceamericano":4500, "americanoice":4500,
           "hotamericano":4500, "americanohot":4500, "americano":4500,
           "icecafelatte":5000, "cafelatteice":5000,
           "hotcafelatte":5000, "cafelattehot":5000, "cafelatte":5000,
            "anything":4500
           }
    s = 0
    for m in order:
        s += menu[m]
    return s

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> order) {
    int answer = 0;
    
    for (const string& menu : order) {
        if (menu.find("cafelatte") != string::npos) {
            answer += 5000;
        }
        else {
            answer += 4500;
        }
    }
    return answer;
}